src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs

```diff
@@ -47,5 +47,23 @@ public static bool SerializationGuard
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get => GetCachedSwitchValue("Switch.System.Runtime.Serialization.SerializationGuard", ref s_serializationGuard);
         }
+
+        private static int s_showILOffset;
+        private static bool GetDefaultShowILOffsetSetting()
+        {
+            if (s_showILOffset < 0) return false;
+            if (s_showILOffset > 0) return true;
+
+            bool isSwitchEnabled = AppContextConfigHelper.GetBooleanConfig("Switch.System.Diagnostics.StackTrace.ShowILOffsets", "DOTNET_ILOffsetToStackTrace");
```

Comment:
Lets align the env var with conventions elsewhere ([example](https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs#L252))
```suggestion
            bool isSwitchEnabled = AppContextConfigHelper.GetBooleanConfig("Switch.System.Diagnostics.StackTrace.ShowILOffsets", "DOTNET_SYSTEM_DIAGNOSTICS_STACKTRACE_SHOWILOFFSETS");
```